app/Services/AiracService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Services;
+
+use Carbon\Carbon;
+
+class AiracService
+{
+    public static function getBaseAiracDate(): Carbon
+    {
+        return Carbon::parse('2021-01-28 00:00:00');
```

Comment:
Will this ever need to change?

Reply:
Shouldn't do unless the AIRAC people change the cycle frequency, just needs to be a valid airac day :)